src/Umbraco.Infrastructure/Telemetry/Providers/SystemInformationTelemetryProvider.cs

```diff
@@ -111,6 +111,15 @@ internal class SystemInformationTelemetryProvider : IDetailedTelemetryProvider,
             { "Current Server Role", CurrentServerRole },
         };
 
+    public IDictionary<string, object> GetServerInformation() =>
```

Comment:
IMO Serverinformation is different from Troubleshooting info. I think we should have an explicit model for server information